### ISSUE TITLE
How much training loss can be reduced？

### ISSUE THREAD

**Issue by Gotta-C** [jwchoi384/Gaussian_YOLOv3#27]
I have trained on my dataset,found the training loss is much higher,it was 40 but ap 98.3.Compared to the original yolov3, the loss is less than 0.3 and ap 99.3.
So  what range of training loss is normal by using new loss func and which tricks you have used?

**Comment by GallonDeng** [comment]
I also found this, the results were not better than the original yolov3 for my dataset

**Comment by jwchoi384** [comment]
@Gotta-C @AllenDun 

In training, please check the mAP rather than loss value. Loss value is just the value related to delta.
https://github.com/jwchoi384/Gaussian_YOLOv3/issues/19#issuecomment-553878638

I did an experiment with only the official published dataset.
Unfortunately, I don't know the cause of the performance drop in your dataset. 
There may be deviations depending on the dataset.
